website/content/en/status/report-2023-04-2023-06/core.adoc

```diff
@@ -6,35 +6,38 @@ The FreeBSD Core Team is the governing body of FreeBSD.
 
 ==== DevSummit 202305
 
-The Core Team has presented the status update at the DevSummit 202305.
-Slides are available at link:https://wiki.freebsd.org/DevSummit/202305[]
+The Core Team has presented the status update at the FreeBSD Developer Summit, 17th–18th May.
+Slides are available at link:https://wiki.freebsd.org/DevSummit/202305[].
 
 ==== FreeBSD 14
 
-The Core Team is working with the different teams to ensure that FreeBSD 14 will come out with the highest quality.
+The Core Team is working with other teams to ensure that FreeBSD 14.0-RELEASE will be of the highest quality.
 
-The Core Team has no objection to mark riscv64sf as unsupported in 14.
+The Core Team has no objection to mark riscv64sf (64-bit RISC-V soft-float) as https://www.freebsd.org/platforms/[unsupported in 14].
 
-==== Meetings with the FreeBSD Foundation
+==== Meetings with The FreeBSD Foundation
 
-The Core Team and the FreeBSD Foundation continue to meet regularly to discuss the next steps to take for the management and development of FreeBSD and its future.
-In 2023Q2, the Core Team had two meetings with the Boards of Directors of the FreeBSD Foundation and employees.
-They discussed how the FreeBSD Foundation can help the Core Team and the Project in general.
+The Core Team and The FreeBSD Foundation continue to meet regularly to discuss the next steps to take for the management, development, and future of FreeBSD.
+The Core Team had two meetings with the Board of Directors of, and employees of, the Foundation.
+They discussed how the Foundation can help the Core Team and the Project in general.
 
 ==== Matrix IM solution
 
-One of the major items in the Core Team updates in DevSummit 202305 is proposing a new project communication channel.
+One of the major items in the Core Team updates in DevSummit 202305 was proposing a new project communication medium.
 
-There is currently a testing instance at matrix-dev.FreeBSD.org setup by clusteradm.
-All developers can access it with their kerberos credentials, and some public channels can be joined through Matrix's federation feature.
-Please note this instance is for testing and evaluating so no backup and availability is guaranteed.
+There is currently a testing instance at matrix-dev.FreeBSD.org setup by clusteradm. The lobby: 
 
-The Core Team is still discussing the scope and administration of this service, and collecting the feedback from community.
+- https://matrix.to/#/#lobby:matrix-dev.freebsd.org[]
+
+All developers can access the instance with their kerberos credentials, and some public rooms can be joined through Matrix's federation feature.
```

Comment:
> … some public rooms … federation …

Technically: is it possible for any truly public room to be explicitly **not** federated?

(I'll seek clarity in a Matrix area.)

Reply:
From <https://matrix.to/#/!OGEhHVWSdvArJzumhm:matrix.org/$usQ0v6T17uMyGebyBFkg2Xpl5VQ-GvTHnFFwMtDZubE?via=matrix.org&via=libera.chat&via=wolfo.tech>: 

> You can make a room publicly joinable on your homeserver without allowing it to be federated.
> 
> Maybe it's not "public" conceptually, but that's how Matrix describes a room that doesn't require an invite to join.